collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -1618,7 +1618,7 @@ private void showCustomToast(String message, int duration) {
      * repeat of the current group.
      */
     private void createRepeatDialog() {
-        FormController formController = Collect.getInstance()
+        final FormController formController = Collect.getInstance()
```

Comment:
I'd tend to move this down to line 1707 since it really doesn't make much sense way up here.